apps/docs/migration/from-zep.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: "Migrating from Zep AI to Supermemory"
+description: "Quick guide to migrate from Zep AI to Supermemory"
+sidebarTitle: "From Zep AI"
+---
+
+## Key Differences
+
+| Zep AI | Supermemory |
+|--------|-------------|
+| Sessions & Messages | Documents & Container Tags |
+| `session.create()` | Use `containerTags` parameter |
+| `memory.add(session_id, ...)` | `memories.add({containerTags: [...]})` |
+| `memory.search(session_id, {text: ...})` | `search.execute({q: ..., containerTags: [...]})` |
+
+## Installation
+
+<CodeGroup>
+
+```bash Python
+pip install supermemory
+```
+
+```bash TypeScript
+npm install supermemory
+```
+
+</CodeGroup>
+
+<CodeGroup>
+
+```python Python
+from supermemory import Supermemory
+client = Supermemory(api_key="your-api-key")
+```
+
+```typescript TypeScript
+import { Supermemory } from "supermemory";
+const client = new Supermemory({ apiKey: "your-api-key" });
+```
+
+</CodeGroup>
+
+## API Mapping
+
+### Session Management
+
+<CodeGroup>
+
+```python Zep AI
+session = client.session.create(
+    session_id="user_123",
+    user_id="user_123"
+)
+```
+
+```python Supermemory
+# No explicit session creation - use containerTags
+container_tags = ["user_123"]
+```
+
+</CodeGroup>
+
+### Adding Memories
+
+<CodeGroup>
+
+```python Zep AI
+client.memory.add(
+    session_id="user_123",
+    memory={"content": "User prefers dark mode"}
+)
+```
+
+```python Supermemory
+client.memories.add({
+    "content": "User prefers dark mode",
+    "container_tags": ["user_123"]
+})
+```
```

Comment:
it's client.add() now  
  
also, it's preferred to use containerTag instead of container_tags